src/schema/v2/startIdentityVerificationMutation.ts

```diff
@@ -0,0 +1,75 @@
+import { mutationWithClientMutationId } from "graphql-relay"
+import { ResolverContext } from "types/graphql"
+import {
+  GraphQLInputObjectType,
+  GraphQLString,
+  GraphQLUnionType,
+  GraphQLObjectType,
+} from "graphql"
+import { GravityMutationErrorType } from "lib/gravityErrorHandler"
+
+const InputType = new GraphQLInputObjectType({
+  name: "StartIdentityVerificationInput",
+  fields: {
+    identityVerificationId: {
+      type: GraphQLString,
+      description: "Primary ID of the identity verification to be started",
+    },
+  },
+})
+
+const FailureType = new GraphQLObjectType<any, ResolverContext>({
+  name: "startIdentityVerificationFailure",
```

Comment:
Types should all be `TitleCase`

Reply:
Done in 263540698f0cfb069ad4ba443f87e375d90590be